Sources/App/Commands/Ingest.swift

```diff
@@ -90,6 +90,7 @@ func ingest(client: Client,
             database: Database,
             logger: Logger,
             packages: [Package]) -> EventLoopFuture<Void> {
+    logger.info("Ingesting \(packages.compactMap {$0.id})")
```

Comment:
I added this additional logging during development but thought it might actually be useful to leave in permanently. It logs the package IDs of all packages about to be ingested.

Reply:
Mmm, I'm not sure how valuable that is in production... We're ingesting 100 packages at a time every two minutes and would get a wall of 100 lines with id that aren't super useful in the logs.

How about we make this a `debug` and then you can use a lower log level for testing?

Reply:
Fixed